### DISCUSSION TITLE
Welcome to t2k4e Discussions!

### DISCUSSION THREAD

**Discussion by Stefouch** [fvtt-fria-ligan/twilight2000-foundry-vtt#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Moonster321** [comment]
Hello!

I am loving T2K for foundry, and have an ongoing campaign that I am a bit worried about with the release of Foundry v13. I was wondering if there are any plans to update the core system and expansions to support V13, or if the V12 update was the final release?

Thank you :)
